src/main/kotlin/hu/kirdev/schpincer/web/DeployController.kt

```diff
@@ -0,0 +1,19 @@
+package hu.kirdev.schpincer.web
+
+import hu.kirdev.schpincer.service.UserService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+open class DeployController {
+
+    @Autowired
+    private lateinit var users: UserService
+
+    @GetMapping("/install/grant-basics")
+    fun grantBasicPermissions(): String {
+        return "" + users.grantAdmin("6af3c8b0-592c-6864-8dd9-5f354edfc0be")
```

Comment:
1. We can use string template instead of concatenatin.
1. This looks like a hash / access token or something like that, shouldn't it be in an environment variable?